src/renderer/components/CreatableSelect.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+import { CSSProperties } from 'react';
+import { StylesConfig } from 'react-select';
+import OriginalCreatableSelect from 'react-select/creatable';
+
+import sassVariables from '../styles/variables.module.scss';
+
+export default function CreatableSelect({
+  controlStyles = {},
+  ...props
+}: {
+  controlStyles: CSSProperties;
+  // TODO: add props typing
```

Comment:
I'm too tired to provide a full solution, but something like this will probably help here:
```ts
type Props = React.ComponentProps<typeof OriginalCreatableSelect>;
```